Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM keydonix/parity-instantseal-node8:safe
+FROM keydonix/parity-instantseal-node8:safe as builder
```

Comment:
What does the `safe` tag mean?

Reply:
This is the hacked tag that used a safer config, (because the other PR i just sent isn't actually shipping the image and i didn't want to override :latest with a hacked together image)

Reply:
`keydonix/parity-instantseal:latest` now has `fast_and_loose: false`, so I think you can switch back to latest once the new `keydonix/parity-instantseal-node8` PR is merged and autodeploys.